Update Doc for ServiceNow to clarify team token configuration guidelines (#914)
hashicorp/terraform-docs-common: website/docs/cloud-docs/integrations/service-now/service-catalog-terraform/index.mdx

3. On the "ServiceNow" team's settings page, generate a [team API
   token](/terraform/cloud-docs/users-teams-organizations/api-tokens#team-api-tokens).
   **Save the team API token for later.**
4. After [generating a team token](/terraform/cloud-docs/users-teams-organizations/api-tokens#team-api-tokens) in HCP Terraform, you can add that token to your ServiceNow context by doing the following:
    1. Navigate to **Terraform Configs** in ServiceNow

>     2. Select 'New' and add token.

Reviewer: We use bold for UI elements:
```suggestion
    2. Select **New** and add your token.
```

begin. You can also refer to this documentation whenever you need to add or
update request items.

### Team Tokens

Adding team-scoped tokens will aid in the management of team access, but is limited within the ServiceNow context. At this time, only one API team token is reccommended, regardless of the the number of teams that will access ServiceNow. 

To add the Team API token that you generated in HCP Terraform:

1. Navigate to Terraform Configs in ServiceNow
2. Select 'New'.
3. Add Token.

> 4. Add org name - this should be the same as your HCP Terraform team name. 

Reviewer: ```suggestion
1. Use your HCP Terraform team name as your **Org name**. 
```

> Note: Provisioning a no-code environment requires the token to have global `READ` access for all projects. Updating configurations will also require `WRITE` access. 

Reviewer: @halie-kastl I believe `WRITE` here would map to the[ project's **Update** role?](https://developer.hashicorp.com/terraform/cloud-docs/users-teams-organizations/permissions#general-project-permissions)
```suggestion
To provision a no-code environment, your team must have [**Read**](https://developer.hashicorp.com/terraform/cloud-docs/users-teams-organizations/permissions#general-project-permissions) access for all projects. To update configurations in no-code environments , you team must have **Update** access. 
```

Reviewer: (Also recommending we don't make this a callout because our style guide advises against them)

begin. You can also refer to this documentation whenever you need to add or
update request items.

### ServiceNow Team Tokens

> Team-scoped tokens can help you manage a team's access to things in HCP Terraform, but team tokens are limited within the ServiceNow context. At this time, only one API team token is reccommended, regardless of the the number of teams that will access ServiceNow. 

Reviewer: @halie-kastl There are still a few typos, so adding a new suggestion:
```suggestion
Team-scoped tokens can help you manage a team's access to things in HCP Terraform, but team tokens are limited within the ServiceNow context. We recommend only using one API team token, regardless of the number of teams accessing ServiceNow. 
```

> Teams,and their associated team tokens, can be assigned to specific projects - [team tokens documentation] (https://developer.hashicorp.com/terraform/cloud-docs/users-teams-organizations/api-tokens#team-api-tokens). Proper permissions are required (e.g. READ, WRITE) to provision a resource. 

Reviewer: According to our style guide, we use inline links as part of sentences so folks have context on where that link will take them. I think this suggestion should cover both:
1. Helping folks understand that different permission levels are necessary for different actions and where to learn more
2. keeping that team tokens information in there but linking to it inline to keep with our style guide! 
```suggestion
You can assign teams and their associated tokens permission levels on specific projects. To learn more about the different permissions necessary to provision resources, refer to [Permissions](/terraform/cloud-docs/users-teams-organizations/permissions#project-permissions). To learn more about API tokens, refer to [Team tokens](https://developer.hashicorp.com/terraform/cloud-docs/users-teams-organizations/api-tokens#team-api-tokens).
```
